### PR TITLE
APPS-314: Added release notes.

### PR DESCRIPTION
**Pull request by anechifor**

### CODE REVIEW

File: docs/Release.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Build
+The `share` project uses _Travis CI_. \
+The `.travis.yml` config file can be found in the root of the repository.
+
+
+## Stages and Jobs
+1. **Build**: Java Build with Unit Tests, WhiteSource, Source Clear Scan (SCA)
+2. **Tests**: Executes the E2E tests in `alfresco-tas-share-test`
+3. **Release**: Release by publishing to Nexus.
+4. **Company Release**: Publish docker image for share to Docker Hub.
```
Comment (cristina-sirbu) [inline docs/Release.md:10]: Which is the step that pushes the Docker image to Quay?

File: docs/Release.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+## Branches
+Travis CI builds differ by branch:
+* `master` / `SP/*` branches:
+  - regular builds which include the _Build_ and _Tests_ stages;
+  - if the commit message contains the `[release]` key word, the builds will also 
+  include the _Release_ stage;
+- if the commit message contains the `[trigger company release]` as commit message, the builds will also
```
Comment (cristina-sirbu) [inline docs/Release.md:18]: This bullet point shouldn't be under ``master` / `SP/*` branches:` ?

File: docs/Release.md (hunk 1 above)
Comment (anechifor) [reply docs/Release.md:10]: In the build stage, I will add the note.

**Comment by cristina-sirbu** [comment]
Can we add a note somewhere that these instructions are only for Branches in this repo and are not available for forks?

**Pull request merged**